kconfiglib.py

```diff
@@ -995,6 +995,50 @@ def _load_config(self, filename, replace):
                 if not choice._was_set:
                     choice.unset_value()
 
+    def _node_items(self):
```

Comment:
Might be nice to have this available in the public API as well.

What do you think of calling the function `node_iter(skip_duplicates=False)`? `skip_duplicates=True` would use the existing logic, while `skip_duplicates=False` would return all nodes.